ttnctl/cmd/components_info.go

```diff
@@ -38,6 +38,19 @@ Created:        2016-10-06 09:52:28.766 +0000 UTC
 		fmt.Printf("Type:           %s\n", component.Type)
 		fmt.Printf("Created:        %s\n", component.Created)
 		fmt.Println()
+		fmt.Println("Collaborators:")
+		for _, collaborator := range component.Collaborators {
+			fmt.Printf("       - Name: %s\n", collaborator.Username)
+			fmt.Print("         Rights: ")
+			for i, right := range collaborator.Rights {
+				if i != 0 {
+					fmt.Print(", ")
+				}
+				fmt.Print(right)
```

Comment:
OK. Would have been nice to have had a `type Rights []Right` then. I can imagine many quick operations to check rights, add right, remove right, format rights, etc. Can add that to https://github.com/TheThingsNetwork/go-account-lib/issues/63. Anyway, leave it here for now